### PR TITLE
Protected all non-auth routes

### PR DESCRIPTION
**Pull request by siddharthnandy**
Protected all routes besides those used for admin/user sign-up and sign-in using passport and JWT.

### CODE REVIEW

**Comment by siddharthnandy** [comment]
Ignore the conversion; I thought the new routes that I made for the password reset stuff would need to be protected but I don't think they will be.

File: mobile/Screen/ArtworkDetailScreen.js [hunk 1]
```diff
@@ -78,31 +84,38 @@ function ArtworkDetailScreen({
     }
   };
 
-  const addSavedArt = async (artId) => {
+  const addSavedArt = async () => {
     try {
-      const res = await axios.patch(`${URL}/user/addArtwork/63e33e2f578ad1d80bd2a347`, [artId]);
+      console.log({ id, render: true });
```
Comment (he-is-james) [inline mobile/Screen/ArtworkDetailScreen.js:89]: Remove unnecessary `console.log`

File: mobile/Screen/EventDetailScreen.js [hunk 2]
```diff
@@ -43,10 +44,13 @@ function EventDetailScreen({
   } = route.params;
 
   const [isEventSaved, setIsEventSaved] = useState(false);
+  const { authHeader } = useSelector((state) => state.auth);
 
   const getSomeEvents = async () => {
     try {
-      const res = await axios.get(`${URL}/user/getEvents/63e33e2f578ad1d80bd2a347`);
+      const res = await axios.get(`${URL}/user/getEvents/63e33e2f578ad1d80bd2a347`, {
```
Comment (he-is-james) [inline mobile/Screen/EventDetailScreen.js:51]: these IDs should be the user ids

File: mobile/Screen/EventDetailScreen.js [hunk 3]
```diff
@@ -43,10 +44,13 @@ function EventDetailScreen({
   } = route.params;
 
   const [isEventSaved, setIsEventSaved] = useState(false);
+  const { authHeader } = useSelector((state) => state.auth);
 
   const getSomeEvents = async () => {
```
Comment (he-is-james) [inline mobile/Screen/EventDetailScreen.js:49]: shouldn't this be named saved events

**Review by he-is-james — APPROVED** [review]
Great overall, just slide changes needed for naming conventions

**Pull request merged**